relay-general/src/protocol/event.rs

```diff
@@ -227,7 +227,7 @@ pub struct Event {
     pub culprit: Annotated<String>,
 
     /// Transaction name of the event.
-    #[metastructure(max_chars = "culprit")]
+    #[metastructure(max_chars = "culprit", trim_whitespace = "true")]
```

Comment:
Should we set `required="true"` and `empty="false"` here? This would emit errors if the name is invalid which show in the UI. You can still default the name in the processor in addition to that.

Reply:
I think the decision is that it is not `required`, cc @HazAT

Are there UI alerts from transactions too? Or just error events?

Reply:
Thanks for clarifying. If it's valid that the name is empty, then we should of course leave it like this.

We would always render the name with hover, and I think we also show the error box in the Discover details view.